src/circuit/freeze.rs

```diff
@@ -30,9 +30,9 @@ use jf_plonk::{
 use jf_primitives::circuit::merkle_tree::AccMemberWitnessVar;
 use jf_utils::fr_to_fq;
 
-pub(crate) struct FreezeCircuit(pub(crate) PlonkCircuit<BaseField>);
+pub(crate) struct FreezeCircuit<C: CapConfig>(pub(crate) PlonkCircuit<C::ScalarField>);
```

Comment:
Do you remember the reason why we used `BaseField` before?

Reply:
I have no idea, I think it was a bad choice judging from now. can be confusing sometimes.

If I have to guess, maybe we try to distinguish `JubjubScalar` from `BLSScalar`, without referring to `BLS` in the name since we also instantiate with `BN`, therefore we choose to use `BaseField` at the time.